src/gap_filler/Shak scratch file.py

```diff
@@ -0,0 +1,11 @@
+
+#variables - the task
+
+sentence_1 = ("I am in the park.")
+sentence_2 = ("in the park.")
+sentence_3 = ("it was in.")
+
```

Comment:
Also to make it even more realistic make the following sentence into a list of words but this time only replace the second 'in' leaving the rest alone. This is very important since in our code we do not want to remove all the words of interest (if you remove all the nouns you can't read anything, as we discovered when we were building the code lol)

New sentence "I am in love with being in the park in my favourite shoes"